test/src/test/java/io/seata/core/rpc/netty/TmNettyClientTest.java

```
@@ -47,17 +47,20 @@
      */
     @Test
     public void testDoConnect() throws Exception {
-
+        ThreadPoolExecutor workingThreads = initMessageExecutor();
+        NettyRemotingServer nettyRemotingServer = new NettyRemotingServer(workingThreads);
         //start services server first
-        workingThreads.submit(new Runnable() {
```

exeucte will swallow the exception